src/go/rpk/pkg/cli/cmd/cluster/config/export/export.go

```diff
@@ -0,0 +1,205 @@
+) (err error) {
+	template := make(map[string]interface{})
+
+	// Present properties in alphabetical order, providing some pseudo-grouping based on common prefixes
+	keys := make([]string, 0, len(schema))
+	for k := range schema {
+		keys = append(keys, k)
+	}
+	sort.Strings(keys)
+
+	yamlNode := yaml.Node{
+		Kind: yaml.MappingNode,
+	}
+
+	for _, name := range keys {
+		meta := schema[name]
+		curValue := config[name]
+		template[name] = curValue
+
+		visibility := meta.Visibility
+
+		if visibility == "deprecated" {
+			continue
+		}
+
+		if meta.Visibility == "tunable" && !all {
+			continue
+		}
+
+		property_node := yaml.Node{
+			Kind:  yaml.ScalarNode,
+			Value: name,
+		}
+
+		var valueNode yaml.Node
+		switch x := curValue.(type) {
+		case int:
+			valueNode = yaml.Node{
+				Kind:  yaml.ScalarNode,
+				Value: fmt.Sprintf("%d", x),
+			}
+		case float64:
+			if x == float64(int64(x)) {
+				valueNode = yaml.Node{
+					Kind:  yaml.ScalarNode,
+					Value: fmt.Sprintf("%d", int64(x)),
+				}
+			} else {
+				valueNode = yaml.Node{
+					Kind:  yaml.ScalarNode,
+					Value: fmt.Sprintf("%f", x),
+				}
+			}
+		case string:
+			valueNode = yaml.Node{
+				Kind:  yaml.ScalarNode,
+				Value: x,
+			}
+		case bool:
+			var x_str string
+			if x {
+				x_str = "true"
+			} else {
+				x_str = "false"
+			}
+			valueNode = yaml.Node{
+				Kind:  yaml.ScalarNode,
+				Value: x_str,
+			}
+		case []interface{}:
+			valueNode = yaml.Node{
+				Kind:  yaml.SequenceNode,
+				Value: "",
+			}
+			// TODO generalize for other types of array
+			for _, v := range x {
+				seqNode := yaml.Node{
+					Kind:  yaml.ScalarNode,
+					Value: v.(string),
+				}
+				valueNode.Content = append(valueNode.Content, &seqNode)
+			}
+		case nil:
+			valueNode = yaml.Node{
+				Kind:  yaml.ScalarNode,
+				Value: "",
+			}
+		default:
+			log.Warningf("Unexpected property value type: %s: %T", name, curValue)
+			continue
+		}
+
+		// TODO: nullability in description
+		// TODO: needs-restart in description
+
+		// Preface each property with a descriptive comment
+		if meta.Example != "" {
+			property_node.HeadComment = fmt.Sprintf("%s (e.g. '%s')", meta.Description, meta.Example)
+		} else {
+			property_node.HeadComment = meta.Description
+		}
+
+		// Blank line between each property for readability
+		property_node.FootComment = "\n"
+
+		yamlNode.Content = append(yamlNode.Content, &property_node)
+		yamlNode.Content = append(yamlNode.Content, &valueNode)
+	}
+
+	serialized, err := yaml.Marshal(&yamlNode)
+	if err != nil {
+		return err
+	}
+
+	_, err = file.Write(serialized)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func NewCommand(fs afero.Fs, all *bool) *cobra.Command {
+	var filename string
+
+	cmd := &cobra.Command{
+		Use:   "export",
+		Short: "Export cluster configuration.",
+		Long: `Export cluster-wide configuration properties
+
+Cluster-wide properties apply to all nodes in a redpanda cluster.  For
```

Comment:
Seems like this help text could be the long help text on `rpk cluster config` itself, and then the per-command help text could explain a bit more about what import/export/etc are doing & why they'd be used.